Exception ignored in: <LazySettings [Unevaluated]>
Traceback (most recent call last):
  File "/usr/lib/python3.10/abc.py", line 119, in __instancecheck__
    return _abc_instancecheck(cls, instance)
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/django/utils/functional.py", line 295, in __getattribute__
    value = super().__getattribute__(name)
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/django/utils/functional.py", line 266, in inner
    self._setup()
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/django/conf/__init__.py", line 82, in _setup
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested settings, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
Traceback (most recent call last):
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/bin/daphne", line 8, in <module>
    sys.exit(CommandLineInterface.entrypoint())
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/daphne/cli.py", line 171, in entrypoint
    cls().run(sys.argv[1:])
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/daphne/cli.py", line 233, in run
    application = import_by_path(args.application)
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/daphne/utils.py", line 12, in import_by_path
    target = importlib.import_module(module_path)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ivan/E/ninja/backend/./backend/asgi.py", line 7, in <module>
    from backend.urls import websocket_urlpatterns
  File "/home/ivan/E/ninja/backend/./backend/urls.py", line 3, in <module>
    from ninja import NinjaAPI
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/__init__.py", line 10, in <module>
    from ninja.main import NinjaAPI
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/main.py", line 23, in <module>
    from ninja.openapi import get_schema
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/openapi/__init__.py", line 1, in <module>
    from ninja.openapi.schema import get_schema
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/openapi/schema.py", line 9, in <module>
    from ninja.operation import Operation
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/operation.py", line 26, in <module>
    from ninja.throttling import BaseThrottle
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/throttling.py", line 50, in <module>
    class SimpleRateThrottle(BaseThrottle):
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/throttling.py", line 63, in SimpleRateThrottle
    from ninja.conf import settings
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/ninja/conf.py", line 31, in <module>
    settings = Settings.model_validate(django_settings)
  File "/home/ivan/.cache/pypoetry/virtualenvs/ninja-M7ErS_rb-py3.10/lib/python3.10/site-packages/pydantic/main.py", line 596, in model_validate
    return cls.__pydantic_validator__.validate_python(
pydantic_core._pydantic_core.ValidationError: 5 validation errors for Settings
PAGINATION_CLASS
  Error extracting attribute: ImproperlyConfigured: Requested setting NINJA_PAGINATION_CLASS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings. [type=get_attribute_error, input_value=<LazySettings [Unevaluated]>, input_type=LazySettings]
    For further information visit https://errors.pydantic.dev/2.9/v/get_attribute_error
PAGINATION_PER_PAGE
  Error extracting attribute: ImproperlyConfigured: Requested setting NINJA_PAGINATION_PER_PAGE, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings. [type=get_attribute_error, input_value=<LazySettings [Unevaluated]>, input_type=LazySettings]
    For further information visit https://errors.pydantic.dev/2.9/v/get_attribute_error
PAGINATION_MAX_LIMIT
  Error extracting attribute: ImproperlyConfigured: Requested setting NINJA_PAGINATION_MAX_LIMIT, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings. [type=get_attribute_error, input_value=<LazySettings [Unevaluated]>, input_type=LazySettings]
    For further information visit https://errors.pydantic.dev/2.9/v/get_attribute_error
NUM_PROXIES
  Error extracting attribute: ImproperlyConfigured: Requested setting NINJA_NUM_PROXIES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings. [type=get_attribute_error, input_value=<LazySettings [Unevaluated]>, input_type=LazySettings]
    For further information visit https://errors.pydantic.dev/2.9/v/get_attribute_error
DEFAULT_THROTTLE_RATES
  Error extracting attribute: ImproperlyConfigured: Requested setting NINJA_DEFAULT_THROTTLE_RATES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings. [type=get_attribute_error, input_value=<LazySettings [Unevaluated]>, input_type=LazySettings]
    For further information visit https://errors.pydantic.dev/2.9/v/get_attribute_error
